flashinfer_bench/bench/timing.py

```
@@ -38,18 +69,166 @@ def _device_lock(device: str) -> LockType:
+        return ret
+    if return_mode == "all":
+        return times
+    elif return_mode == "min":
+        return min(times)
+    elif return_mode == "max":
+        return max(times)
+    elif return_mode == "mean":
+        return statistics.mean(times)
+    elif return_mode == "median":
+        return statistics.median(times)
+    raise ValueError(f"Unknown return_mode: {return_mode}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)